src/app/tests/TestReadInteraction.cpp

```diff
@@ -287,51 +323,10 @@ void TestReadInteraction::TestReadClientInvalidReport(nlTestSuite * apSuite, voi
     readClient.Shutdown();
 }
 
-void TestReadInteraction::TestReadHandlerInvalidAttributePath(nlTestSuite * apSuite, void * apContext)
```

Comment:
Why remove this too?

Reply:
Accidental, remove, just bring it back, thanks